ruby/vampires.rb

```diff
@@ -1,4 +1,4 @@
-p "How many employees will be processed?"
+  p "How many employees will be processed?"
   number_employees = gets.chomp.to_i
 
 loop_counter = 0

```

Comment:
Line 26

I would like to see you follow the age convention in the instructions.  We want to know if you the employee is lying about their age.

Lets consider checking the current year against how old they say they are and what year they say that they were born in.  Investigate the Ruby `Time` class and see if you can come up with something creative and dynamic.